stack_overflow_page.py

```diff
@@ -26,11 +29,12 @@ def login():
 
         elem = driver.find_element_by_class_name("mini-avatar")
         assert os.environ['STACK_OVERFLOW_DISPLAY_NAME'] in elem.text
+        logging.info("Logged into stackoverflow.com and accessed profile page. ")
         print("Logged into stackoverflow.com and accessed profile page")
```

Comment:
I think we can remove the `print` message here 🙂

Reply:
Whoops! Don't know how I missed that one :sweat_smile: